products/appengine/api.yaml

```diff
@@ -401,8 +402,8 @@ objects:
         name: 'deployment'
         description: |
            Code and application artifacts that make up this version.
-        required: false
-        properties:      
+        required: true
```

Comment:
Tried this myself, looks like leaving it blank returns an API error stating it is required. Thanks for the update